apps/site/src/components/pages/wordpress/header.tsx

```diff
@@ -0,0 +1,150 @@
+import { Stream } from "@cloudflare/stream-react";
+import { Box, Container, Typography, useTheme } from "@mui/material";
+import Image from "next/legacy/image";
+
+import wpHelixImage from "../../../../public/assets/new-home/wp-helix.png";
```

Comment:
This helix image is appearing far more pixelated at large screen widths than that in the background of the current https://blockprotocol.org homepage. _Note: these images are slightly different, but substantially similar._

**Observations:**
- This new image comes in at a much smaller filesize than the original (expected given the lower-quality).
- We have an existing ticket open to optimize the large filesize of the original (so it should not be considered _ideal_).
- However, the trade-off here is probably more than we want given the visual artefacts that are visible at larger screen-sizes.

We should see if we can create a better `webp` or vector original. IIRC the latter will not be possible as this is just a raster I'd exported from Spline, so the former is probably our best bet.

Reply:
I should wait for a higher quality original then, right?

Reply:
We'll do this in a follow-up PR. I'll produce it.